File: C2110iDoiCan/IASF/de02/src/main/java/com/aptech/de02/models/Category.java
package com.aptech.de02.models;
import jakarta.persistence.*;
import lombok.*;

import java.util.List;

@Table(name = "categories")
@Entity
@Builder
@Getter
@Setter
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Category {
    @Id
    @Column(name = "category_id")
    private String categoryId;

    @Column(name = "category_name")
    private String categoryName;

    @Column(name = "description")
    private String description;

    @OneToMany(mappedBy = "category")
    private List<Product> products;

}

File: C2002LDoiCan/EAP/23-10-2021/EAP_C2002L_NguyenVanA/EAP_C2002L_NguyenVanA/Models/Category.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace EAP_C2002L_NguyenVanA.Models
{
    public class Category
    {
        public int CategoryId { get; set; }
        public string CategoryName { get; set; }
        public virtual ICollection<Product> Products { get; set; }
        public Category() {
            Products = new HashSet<Product>();
        }
    }
}